docs/tutorial/az-mysql-tutorial.md

```diff
@@ -0,0 +1,531 @@
+```
+
+Now create a replication user with SSL requirements:
+
+```bash
+# Connect to MySQL with the root password we set earlier
+mysql -u root -p << 'EOF'
+-- Drop user if it exists, then create it
+DROP USER IF EXISTS 'replica'@'%';
+CREATE USER 'replica'@'%' IDENTIFIED BY 'replica_pass' REQUIRE SSL;
+GRANT REPLICATION SLAVE ON *.* TO 'replica'@'%';
+FLUSH PRIVILEGES;
+EOF
+```
+
+Create an initial test database and a table for replication verification:
+
+```bash
+mysql -u root -p -e "CREATE DATABASE testdb; USE testdb; CREATE TABLE t1 (id INT); INSERT INTO t1 VALUES (1);"
+```
+
+## 3. Configure the standby VM as a replica
+
+Access the standby VM, `mysql-standby`, for its configuration:
+
+```bash
+multipass shell mysql-standby
+```
+
+First, configure MySQL authentication on the standby VM to match the primary:
+
+```bash
+# Connect to MySQL as root using sudo (auth_socket) and configure password authentication
+sudo mysql << 'EOF'
+-- Set a password for root user and enable password authentication (same as primary)
+ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY 'SecureRootPass123';
+-- Allow root connections from any host (needed for potential failover scenarios)
+CREATE USER 'root'@'%' IDENTIFIED BY 'SecureRootPass123';
+GRANT ALL PRIVILEGES ON *.* TO 'root'@'%' WITH GRANT OPTION;
+-- Remove anonymous users and test database for security
+DROP USER IF EXISTS ''@'localhost';
+DROP USER IF EXISTS ''@'%';
+DROP DATABASE IF EXISTS test;
+FLUSH PRIVILEGES;
+EOF
+```
+
+**Important**: Use the same root password (`SecureRootPass123`) as configured on the primary server.
+
+Edit the MySQL configuration file /etc/mysql/mysql.conf.d/mysqld.cnf to include these settings:
+
+```ini
+server-id=2
+relay-log=relay-log
+bind-address=0.0.0.0
+
+# SSL configuration for secure replication
+ssl-ca=/etc/mysql/ssl/ca-cert.pem
+ssl-cert=/etc/mysql/ssl/server-cert.pem
+ssl-key=/etc/mysql/ssl/server-key.pem
+```
+
+Restart MySQL after configuration:
+
+```bash
+sudo systemctl restart mysql
+```
+
+From the primary VM, obtain the master status:
+
+```bash
+mysql -u root -p -e "SHOW MASTER STATUS\G"
+```
+
+Take note of:
+
+* File: e.g. mysql-bin.000002
+* Position: e.g. 157
+
+Copy the SSL certificates to the standby VM. First, on the primary VM, copy certificates to a location that can be transferred:
+
+```bash
+# On mysql-primary, copy certificates to home directory for transfer
+sudo cp /etc/mysql/ssl/ca-cert.pem /etc/mysql/ssl/client-cert.pem /etc/mysql/ssl/client-key.pem ~/
+sudo chown ubuntu:ubuntu ~/ca-cert.pem ~/client-cert.pem ~/client-key.pem
+```
+
+Now transfer the certificates from your host machine:
+
+```bash
+multipass transfer mysql-primary:ca-cert.pem .
+multipass transfer ca-cert.pem mysql-standby:
+multipass transfer mysql-primary:client-cert.pem .
+multipass transfer client-cert.pem mysql-standby:
+multipass transfer mysql-primary:client-key.pem .
+multipass transfer client-key.pem mysql-standby:
+```
```

Comment:
Should we add an additional command to remove the certificates from the host? Just to be tidy and to not encourage users to leave their certificates lying around.